applications/FluidDynamicsApplication/tests/two_fluid_mass_conservation_source_test.py

```diff
@@ -0,0 +1,160 @@
+            "python_module" : "json_output_process",
+            "kratos_module" : "KratosMultiphysics",
+            "process_name"  : "JsonOutputProcess",
+            "Parameters"    : {
+                "output_variables" : ["DISTANCE"],
+                "output_file_name" : "TO_BE_DEFINED",
+                "model_part_name"  : "FluidModelPart.FluidParts_Boundaries-Fluid",
+                "time_frequency"   : 0.01
+            }
+        }""")
+        output_file_name = "mass_conservation_{0}d_results.json".format(dim)
+        json_output_settings["Parameters"]["output_file_name"].SetString(output_file_name)
+        self.parameters["processes"]["json_check_process_list"].Append(json_output_settings)
+
+    def _AddReferenceValuesCheck(self):
+        dim = self.parameters["solver_settings"]["domain_size"].GetInt()
+        json_check_settings = KratosMultiphysics.Parameters("""{
+            "python_module" : "from_json_check_result_process",
+            "kratos_module" : "KratosMultiphysics",
+            "process_name"  : "FromJsonCheckResultProcess",
+            "Parameters"    : {
+                "check_variables"      : ["DISTANCE"],
+                "input_file_name"      : "TO_BE_DEFINED",
+                "model_part_name"      : "FluidModelPart.FluidParts_Boundaries-Fluid",
+                "tolerance"            : 0.0,
+                "relative_tolerance"   : 0.0,
+                "time_frequency"       : 0.01
+            }
+        }""")
+        input_file_name = "mass_conservation_{0}d_results.json".format(dim)
+        json_check_settings["Parameters"]["input_file_name"].SetString(input_file_name)
+        json_check_settings["Parameters"]["tolerance"].SetDouble(self.check_absolute_tolerance)
+        json_check_settings["Parameters"]["relative_tolerance"].SetDouble(self.check_relative_tolerance)
+        self.parameters["processes"]["json_check_process_list"].Append(json_check_settings)
+
+class FluidDynamicsAnalysisSloshingTank(FluidDynamicsAnalysis):
+
+    def __init__(self,model,project_parameters,flush_frequency=10.0):
+        super().__init__(model,project_parameters)
+        self.flush_frequency = flush_frequency
+        self.last_flush = time.time()
+
+    def ModifyInitialGeometry(self):
+
+        h0=0.5
+        hmax=0.1
+        for node in self._GetSolver().GetComputingModelPart().Nodes:            
+            d0=node.Y-(h0+hmax*math.cos(math.pi*(1-node.X)))
+            node.SetSolutionStepValue(KratosMultiphysics.DISTANCE,d0)
+
+    def FinalizeSolutionStep(self):
+        super().FinalizeSolutionStep()
+        if self.parallel_type == "OpenMP":
+            now = time.time()
+            if now - self.last_flush > self.flush_frequency:
+                sys.stdout.flush()
+                self.last_flush = now
+
+if __name__ == "__main__":
+
+    test = TwoFluidMassConservationTest()
+
+    test.runTwoFluidMassConservationTest2D()
+
+    #test.runTwoFluidMassConservationTest3D()
```

Comment:
```suggestion
    KratosUnittest.main()
```
By doing this you ensure that all the tests in the file are run.